osgp/shared/osgp-throttling-client/src/main/java/org/opensmartgridplatform/throttling/ThrottlingClient.java

```diff
@@ -121,6 +148,11 @@ private void registerClient() {
 
   /** Lets the Throttling REST service know this client is going away. */
   public void unregister() {
+    if (this.clientId == null) {
+      LOGGER.info("Client was not registered yet, so skip unregistration");
```

Comment:
The comment certainly applies to the most common case when unregistration is skipped.
Because the `clientId` can also be `null` if `unregister` is called multiple times, it might be more accurate, to only mention that the `ThrottlingClient` does not have a registered `clientId` without the yet.

Reply:
Changed